README.md

```diff
@@ -1,14 +1,13 @@
 # Universal Connect Widget (Application)
```

Comment:
Some simple typo fixes, and added more info about Docker publishing, as well as some additional details to how to run docker, and potential issues that might arise.